src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -157,6 +159,21 @@ This resets the cluster to its initial state.`,
 		Short: "Forward a port on the local machine to pachd. This command blocks.",
 		Long:  "Forward a port on the local machine to pachd. This command blocks.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			pidfile.SetPidfilePath("~/.pachyderm.port.forward.pid")
```

Comment:
Generally I think we're putting any files we need for state under `~/.pachyderm` ... as defined in the `/src/client/pkg/config` package. We should probably be consistent w that code and do something like `~/.pachyderm/port-forward.pid`